create_infotexts.py

```diff
@@ -258,19 +258,13 @@ def create_commons_filename(self, item):
             fname_desc += " {}".format(item["Land, foto"])
 
         if fname_desc.endswith(","):
-            fname_desc = "{}".format(fname_desc.strip(","))
+            fname_desc = "{},".format(fname_desc.strip(","))
```

Comment:
Clarification to https://github.com/mattiasostmar/SMVK-Cypern_2017-01/pull/32#discussion_r114496975

You want each part (ort, region, land etc.) to be comma separated. So you want a comma after the `{}` on L252, L255, L258.

Then you want to ensure that the whole ort, region, land chain is ended by a full stop (if any of them where added at all). That is why you look for the comma and if found replaces it by a full stop.

In you look at your current implementation on L260-261 it actually doesn't do anything at all!
What you want is a `{}.` here instead of `{},`.